images/php-fpm/Dockerfile

```diff
@@ -33,7 +33,10 @@ RUN set -xe; \
         /var/lib/apt/lists/* \
         /usr/src/php/ext/* \
         /tmp/*
-
+RUN apt-get -yqq install exiftool
```

Comment:
This should be moved up to join the rest of the extension installs. Each `RUN` instruction creates a new layer of this Docker image. We want to minimize unnecessary layers.